src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const workingState = { ...state };
```

Comment:
[NAMING] - use proper names for object copy. Instead of `workingState`, a name like `stateCopy` would be more descriptive and adhere to best practices.